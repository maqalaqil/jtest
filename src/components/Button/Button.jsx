import React from 'react'

export default function Button() {
    return (
        <div>
            <button>press here</button>
        </div>
    )
}
