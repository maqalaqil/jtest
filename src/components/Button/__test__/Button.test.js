import * as React from "react";
import * as renderer from "react-test-renderer";
import Button from '../Button'

it("this is working head", () => {
  const tree = renderer.create(<Button />).toJSON();
  expect(tree).toMatchSnapshot();
});