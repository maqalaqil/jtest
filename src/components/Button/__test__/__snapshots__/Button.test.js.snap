// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`this is working head 1`] = `
<div>
  <button>
    press here
  </button>
</div>
`;
